internal/pkg/core/plugin/process/start.go

```diff
@@ -168,14 +174,14 @@ func injectLogLevel(logLevel string, args []string) []string {
 	var level string
 	// Translate to level beat understands
 	switch logLevel {
-	case "info":
-		level = "info"
-	case "debug":
-		level = "debug"
-	case "warning":
-		level = "warning"
-	case "error":
-		level = "error"
+	case levelInfo:
+		level = levelInfo
+	case levelDebug:
+		level = levelDebug
+	case levelWarning:
+		level = levelWarning
+	case levelError:
+		level = levelError
```

Comment:
[Suggestion]
what is the point of this `switch`? Without a `default`, better just do `level = logLevel`. Even with a default case, I's suggest to simplify like

```suggestion
	case levelInfo:
	case levelDebug:
	case levelWarning:
	case levelError:
		level = logLevel
	default:
		return args
```